rbac/internal/views.py

```diff
@@ -659,6 +660,89 @@ def list_or_delete_bindings_for_role(request, role_uuid):
         return HttpResponse(json.dumps(info), status=200)
 
 
+def clean_binding_mapping(request, binding_id):
+    """Clean bindingmapping for a role, delete not associated role anymore.
+
+    POST /_private/api/utils/bindings/<binding_id>/clean
+    Params:
+        users: comma separated user ids
+        groups: comma seprated group uuids
+    """
+    if not destructive_ok("api"):
+        return HttpResponse("Destructive operations disallowed.", status=400)
+    if request.method != "POST":
+        return HttpResponse('Invalid method, only "POST" is allowed.', status=405)
+    users = request.GET.get("users")
+    groups = request.GET.get("groups")
+    if not users and not groups:
+        return HttpResponse(
+            'Invalid request, must supply the "users" or "groups" in body.',
+            status=400,
+        )
+
+    replicator = OutboxReplicator()
+    with transaction.atomic():
+        mapping = (
+            BindingMapping.objects.select_for_update()
+            .filter(
+                id=binding_id,
+            )
+            .get()
+        )
+        if users:
+            users = users.split(",")
+            relations_to_remove = []
+            # Check if the user should be removed
+            assert (
```

Comment:
can we return http response with related message to see what is error when we will call endpoint ?
(I am not sure what we would see on assert error)